<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicView</class>
 <widget class="QWidget" name="MusicView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>844</width>
    <height>426</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>vkDownloader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/new/prefix1/1400157620_icon-pause.png</normalon>
   </iconset>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>431</width>
     <height>371</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Это список ваших треков. Если там пусто,а у вас есть треки, просто подождите пока они загрузятся.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
   <property name="layoutMode">
    <enum>QListView::SinglePass</enum>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="setPath">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>114</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Путь</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во песен</string>
   </property>
  </widget>
  <widget class="QLabel" name="countSongs">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="allProgress">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>361</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Общий прогресс</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="curProgBar">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Текущая песня</string>
   </property>
  </widget>
  <widget class="QLabel" name="current">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="currentSong">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>140</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DOWNLOAD</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentSongSpeed">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Player</string>
   </property>
   <widget class="QSlider" name="curPlayer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/new/prefix1/1400113331_icon-ios7-arrow-back.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play_pause">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/1400113310_icon-play.png</normaloff>
      <normalon>:/new/prefix1/1400113310_icon-play.png</normalon>../../../../Downloads/1400113310_icon-play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>45</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="default.qrc">
      <normaloff>:/new/prefix1/1400113325_icon-ios7-arrow-forward.png</normaloff>:/new/prefix1/1400113325_icon-ios7-arrow-forward.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="curSongTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="curPlaySong">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>391</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Помощь</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Скачивание одного трека: выделить его и нажать Cmd+S&lt;/p&gt;&lt;p&gt;Для остальных просто выделить и нажать DOWNLOAD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>811</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DOWNLOAD</string>
   </property>
  </widget>
  <action name="action">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Скачка одной</string>
   </property>
   <property name="toolTip">
    <string>Будет скачиваться одна песня</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S, Meta+S, Meta+Ы</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="default.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>signal1()</signal>
 </slots>
</ui>
